sources/packages/web/src/views/PersonalInfoQuestionnaire.vue

```diff
@@ -149,7 +149,7 @@
       </Section>
       <HorizontalSeparator />
     </div>
-    <Button @click="onNext" class="p-m-6">Next</Button>
+    <Button @click="onNext" class="p-m-6">Next Section</Button>
```

Comment:
Are we not supposed to have a Previous Section button too as per the figma?

Reply:
@2011sandeepraj , I will added that as per your recommendation but in SIM-160, there is no specific acceptance criteria for this.

Reply:
Thanks. Once added, consider the PR approved. Please go ahead with merge to main.